PF/Assignment3/Assignment1a.java

```diff
@@ -0,0 +1,41 @@
+package Assignment3;
+
+import java.util.Scanner;
+
+//this is the class for finding the remainder. Here remainder is calculated using recursion
+public class Assignment1a {
+
+	public static void main(String[] args) {
+		Assignment1a remain = new Assignment1a();
+		Scanner sc = new Scanner(System.in);
+		System.out.println("enter Dividend");
+		int dividend = sc.nextInt(); // variable for dividend value
+		System.out.println("enter Divisor");
+		int divisor = sc.nextInt(); // variable for divisor value
+		int remainder = remain.rem(dividend, divisor);
+		if (remainder == -1)
+			System.out.println("Remainder is not possible");
+		else
+			System.out.println("Remainder is" + remainder);
+	}
+
+	int rem(int x, int y) // function for finding remainder. The function is
```

Comment:
Comment the method and class like below and parameter as well as return type description:
/*
*   function for finding remainder. The function using recursion
* @param int x to get dividend
* @param int y to get divisor
*  @return int gives the remainder value
*/
int rem(int x, int y)